src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-student-comp {
+  height: 316.9661865234375px;
```

Comment:
You should start by removing width/height and then adjusting `max-width` and `max-height` until your Storybook looks nice.
The three files look vastly different from each other. For example, `student.png` is extremely huge compared to the other two files!